python/dgl/graphbolt/link_data_format.py

```diff
@@ -0,0 +1,23 @@
+"""Linked data format."""
+
+from enum import Enum
+
+__all__ = ["LinkDataFormat"]
+
+
+class LinkDataFormat(Enum):
+    """
+    An Enum class representing the two data formats used in link prediction:
```

Comment:
If we are 100% sure there are 2 data formats, then we should use boolean, if we use enum, which means there are chances to extend this, then don't mention 2 data formats.
```suggestion
    An Enum class representing the data formats used in link prediction:
```